angular.json

```diff
@@ -976,7 +988,47 @@
           }
         }
       }
+    },
+    "customer-ticketing": {
+      "projectType": "library",
+      "root": "feature-libs/customer-ticketing",
+      "sourceRoot": "feature-libs/customer-ticketing",
+      "prefix": "cx",
+      "architect": {
+        "build": {
+          "builder": "./tools/build-lib:augmented-types",
+          "options": {
+            "tsConfig": "feature-libs/customer-ticketing/tsconfig.lib.json",
+            "project": "feature-libs/customer-ticketing/ng-package.json"
+          },
+          "configurations": {
+            "production": {
+              "tsConfig": "feature-libs/user/tsconfig.lib.prod.json"
+            }
+          }
+        },
+        "test": {
+          "builder": "@angular-devkit/build-angular:karma",
+          "options": {
+            "main": "feature-libs/customer-ticketing/test.ts",
+            "tsConfig": "feature-libs/customer-ticketing/tsconfig.spec.json",
+            "karmaConfig": "feature-libs/customer-ticketing/karma.conf.js"
+          }
+        },
+        "lint": {
+          "builder": "@angular-devkit/build-angular:tslint",
```

Comment:
[ line 1019 - 1027](https://github.com/SAP/spartacus/pull/15753/files#diff-2d2675bb4687601a5c7ccf707455132f90f3516a33716185687e5c41df59ac7dR1019-R1027)

I don't think this was supposed to be done this way. How did it come to be this way? 

I believe the lint object should have been like the following

```
        "lint": {
          "builder": "@angular-eslint/builder:lint",
          "options": {
            "lintFilePatterns": [
              "feature-libs/customer-ticketing/**/*.ts",
              "feature-libs/customer-ticketing/**/*.html"
            ]
          }
```